proc_windows.go

```diff
@@ -4,9 +4,14 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
-	"syscall"
+
+	"golang.org/x/sys/windows"
 )
 
+const sigint = windows.SIGINT
+const sigterm = windows.SIGTERM
+const sighup = windows.SIGHUP
```

Comment:
Thank you. windows package define SIGHUP but won't work with this since Windows can not handle signals.

Reply:
Fixed in c0477ab